DOMEvents/event-types/events-keyboardevents/KeyboardEvent.location-manual.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>KeyboardEvent.location</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script>
+async_test(function() {
+  var event = "keydown";
+  var expected = ["a", 0, "Control", 1, "Shift", 2];
+  var actual = [];
+
+  window.onload = this.step_func(function() {
+    var target = document.getElementById("target");
+    target.addEventListener(event, this.step_func(function test_event(e) {
```

Comment:
Probably location should be checked in https://github.com/w3c/web-platform-tests/blob/master/uievents/keyboard/key-manual.js and then this would be redundant